docs/user/integration/aws-cloudwatch/aws-otel.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: otel-collector
+data:
+  collector.conf: |
+    receivers:
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: otel-collector
+              scrape_interval: 5s
+              static_configs:
+                - targets: [localhost:8888]
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        limit_mib: 50
+        check_interval: 1s
+      batch/traces:
+        timeout: 1s
+        send_batch_size: 50
+    exporters:
+      awsxray:
+        region: ${AWS_REGION}
+      awsemf:
+        region: ${AWS_REGION}
+        log_retention: 7
```

Comment:
Mention the retention time somewhere in the docs and ask the user to adjust that to their needs.